.github/workflows/reusable-app-prod.yml

```diff
@@ -323,7 +324,9 @@ jobs:
 
     - name: Upload results
       if: always()
-      uses: actions/upload-artifact@v4
+      # Use v3 until the following issues are resolved
+      # see: https://github.com/actions/upload-artifact/issues/478
+      uses: actions/upload-artifact@v3
```

Comment:
複数の job で同一の artifact name でアップロードできなくなったためダウングレード
> Due to how Artifacts are created in this new version, it is no longer possible to upload to the same named Artifact multiple times. You must either split the uploads into multiple Artifacts with different names, or only upload once. Otherwise you will encounter an error.
> https://github.com/actions/download-artifact?tab=readme-ov-file#breaking-changes


https://github.com/actions/download-artifact?tab=readme-ov-file#breaking-changes